templates/index.html

```diff
@@ -1,96 +1,64 @@
 {% extends "base.html" %}
 
 {% block content%}
-    <div id="app-organized-list-container" class="row">
-        <div class="flex-container">
-                <div class="left-col col-md-3 col-xs-6">
-                <div class="side-nav sticky">
-                    <div class="scroll">
-                        <table class="table">
-                            <tbody>
-                            {% for item, app_list in organized_list.items() %}
-                                <tr><td> <a class="anchor-link" href="#{{ item }}">{{ item }}</a></td></tr>
-                            {% endfor %}
-                            </tbody>
-                        </table>
-                        <div class="free-apps">
-                           <a href="https://narrative.kbase.us/#signup">Use these apps with a free KBase account.</a>
-                        </div>
-                    </div>
-                </div>
+<div id="app-organized-list-container" class="row">
```

Comment:
A lot of the indentation in this file was incosistent, plus there was an unclosed div tag and a mismatched `endif`